app/controllers/authentications_controller.rb

```diff
@@ -0,0 +1,25 @@
+class AuthenticationsController < ApplicationController
+
+  include RequireRecentSignin
```

Comment:
Hm also I have to include it in a bunch of handlers and I don't think there's a superclass I can use in that case...